docs/ruleset_functions/get_wwr.md

```diff
@@ -21,29 +23,28 @@ Logic:
 
 - For each building segment in the model: `for building_segment in RMR.building.building_segments:`
 
-  - For each thermal_block in building segment: `for thermal_block in building_segment.thermal_blocks:`
+  - For each thermal block in building segment: `for thermal_block in building_segment.thermal_blocks:`
 
     - For each zone in thermal block: `zone in thermal_block.zones:`
 
-      - Check if zone is conditioned or semi-heated: `if zcc_dictionary[zone.id] in [CONDITIONED RESIDENTIAL, CONDITIONED NON-RESIDENTIAL, CONDITIONED MIXED, SEMI-HEATED]`
+      - Check if zone is conditioned or semi-heated: `if zcc_dictionary[zone.id] in ["CONDITIONED RESIDENTIAL", "CONDITIONED NON-RESIDENTIAL“, ”CONDITIONED MIXED", "SEMI-HEATED"]:`
 
         - For each surface in zone: `for surface in zone.surfaces:`
 
-          - If surface is above-grade wall and adjacent to the exterior: `if ((get_opaque_surface_type(surface) == "ABOVE-GRADE WALL") AND (surface.adjacent_to == "EXTERIOR")):`  
+          - If surface is above-grade wall and adjacent to the exterior: `if ( get_opaque_surface_type(surface) == "ABOVE-GRADE WALL" ) AND ( scc_dictionary[surface.id] in ["EXTERIOR RESIDENTIAL", "EXTERIOR NON-RESIDENTIAL", "EXTERIOR MIXED", "SEMI-EXTERIOR"] ):`  
 
             - For each subsurface in surface: `for subsurface in surface.subsurfaces:`  
 
               - If the glazed vision area is more than 50% of the total area, add glazed vision area to building segment total fenestration area: `if ( subsurface.glazed_area > ( subsurface.glazed_area + subsurface.opaque_area ) * 50% ): total_fenestration_area += subsurface.glazed_area + subsurface.opaque_area`  
 
       - For each surface in zone: `for surface in zone.surfaces:`
 
-        - If surface is above-grade wall and is regulated: `if ((get_opaque_surface_type(surface) == "ABOVE-GRADE WALL") AND (scc_dictionary[surface.id] != "UNREGULATED")):`
+        - If surface is above-grade wall and is regulated: `if ( get_opaque_surface_type(surface) == "ABOVE-GRADE WALL" ) AND ( scc_dictionary[surface.id] != "UNREGULATED" ):`
 
           - Add to envelope total above-grade wall area: `total_envelope_wall_area += surface.area`
 
```

Comment:
I wonder why you use different condition in row 42 vs row 34 above. I think they are equivalent, but it would be more intuitive if they are the same. Also, why not include summing up areas in row right after row #34?

Reply:
This part was not updated accordingly last time when I updated surface.adjacent_to == "EXTERIOR" in the previous loop. This is now modified.